cfgov/unprocessed/css/pages/ask.less

```diff
@@ -270,11 +270,11 @@
     }
 
     .question_summary {
-        padding: unit( (@grid_gutter-width / 2 ) / @base-font-size-px, em ) 
-                 0 
-                 unit( @grid_gutter-width / @base-font-size-px, em ) 
+        padding: unit( (@grid_gutter-width / 2 ) / @base-font-size-px, em )
+                 0
+                 unit( @grid_gutter-width / @base-font-size-px, em )
                  0;
```

Comment:
Not a comment that needs to be addressed here, but the second `0` isn't necessary.